src/Plugin.php

```diff
@@ -59,6 +59,10 @@ public static function init() {
     add_action('rss2_item', __NAMESPACE__ . '\Feed::rss2_item');
 
     UserFrontend::init();
+
+    add_filter('the_content', __CLASS__ . '::the_content');
+    add_filter('wp_get_attachment_metadata', __CLASS__ . '::wp_get_attachment_metadata', 10, 2);
+    add_filter('post_thumbnail_html', __CLASS__ . '::removeSvgWidthHeightAttributes', 10, 3);
```

Comment:
1. There should be an inline comment above this block of filter registrations, explaining the overall feature (i.e., all filters combined).

1. Why is the third callback not named `post_thumbnail_html`?